index.html

```diff
@@ -456,7 +467,8 @@
 			$('.toggleRHP').click(function(){
 			   toggledRHP = !toggledRHP;
```

Comment:
This variable is a  bit confusing. Could you rename it to make the meaning of its values a bit more obvious (when is the pane hidden and when is it shown)?